spec/features/bucket_lists/user_edits_bucket_lists_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+feature 'authenticated user creates a bucket list', %(
+  As a last minute traveler or traveler who's stay is unexpectedly extended,
+  I want to edit a bucket list
+  So I can later recall the destinations without weighing down my bag or mind.
+) do
+# Acceptance Criteria
+#[x] I can change the title and description
+#[x] I must click submit to save my changes
+#[x] I can toggle between private and public
```

Comment:
Incorrect indentation detected (column 0 instead of 2).<br>Missing space after #.